int8 hay_alguien
